unlock-app/src/pages/provider.js

```diff
@@ -17,7 +17,7 @@ export function Web3Provider({ setProvider, config, provider }) {
         <title>{pageTitle('Pick Web3 Provider')}</title>
       </Head>
       <NoSSR>
-        <p>
+        <div>
```

Comment:
I'm ok with this change here because this page is not "user facing" but I think generally everytime we can use semantic elements we should ;) `div` are sad :(